src/modules/cache-client/cache-client.service.ts

```diff
@@ -594,7 +594,22 @@ export class CacheClient implements ICacheClient {
       return false;
     }
 
-    getLogger().debug(`[CACHE CLIENT] axios error: ${error.message}`);
+    const errorDetails = {
+      message: error.message,
```

Comment:
This object can be loged by `inspect` function from utils

Reply:
I think we need to be kind of backward compatible here because this change is implemented at the request of people struggling while debugging now. So, just adding details to this exact error log line is a good starting point. If it gets too noisy we can narrow it in the next iteration.